content/Concepts/commands.mdx

```diff
@@ -1,57 +1,181 @@
 ---
-id: exporting
-slug: exporting
-title: Exporting & Synchronizing Configurations
+slug: commands
+title: Configu's Commands
 ---
 
```

Comment:
For each command, please include links to "additional information" (both CLI and SDK).